openrtb_ext/user.go

```diff
@@ -0,0 +1,13 @@
+package openrtb_ext
+
+// ExtUser defines the contract for bidrequest.user.ext
+type ExtUser struct {
+	DigiTrust *ExtUserDigiTrust `json:"digitrust,omitempty"`
+}
+
+// ExtUserDigiTrust defines the contract for bidrequest.user.ext.digitrust
+type ExtUserDigiTrust struct {
+	ID   string `json:"id"`
+	KeyV int    `json:"keyv"`
+	Pref int    `json:"pref"`
```

Comment:
Can you leave some comments here? `KeyV` and `Pref` aren't descriptive to people who don't know what digitrust is.

Related: I'm not familiar at all with `DigiTrust`... but at a high level, it looks like it's trying to replace Cookie Syncs with a global User ID. Is that User ID one of `ID` or `KeyV` here?

If so, does it duplicate `request.user.id` and/or `request.user.buyeruid`? Ideally we wouldn't do that... but in this case, there might be a strong reason for it. For example, a Pub might run an auction with some bidders--half of whom recognize Digitrust. By defining `user.id` explicitly, it would prevent PBS from respecting cookie syncs.

Might be something I should Slack someone from Product about, to better understand how these things are used.

Reply:
added.